bin/install.sh

```diff
@@ -146,8 +147,11 @@ function install_packages() {
         APT_INSTALL_ARGS+=("network-manager")
     fi
 
-    sudo sed -i 's/apt.screenlyapp.com/archive.raspbian.org/g' \
-        /etc/apt/sources.list
+    if [ "$ARCHITECTURE" != "x86_64" ]; then
```

Comment:
What about x86_32? Better to just look for x86 in the string.

Reply:
Focus will be put on 64-bit x86 devices as most of them already uses 64-bit.